tests/conftest.py

```diff
@@ -52,6 +52,7 @@ def app(request):
     app_ = Flask(__name__, instance_path=instance_path)
     app_.config.update(
         SECRET_KEY='CHANGE_ME',
+        SERVER_NAME='localhost',
```

Comment:
It should not be necessary to include it here. Make sure that you are calling `url_for` inside request context.